misc/loop/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+services:
+  traefik:
+    image: "traefik:v2.10"
+    network_mode: host
+    container_name: "traefik"
+    command:
+      - "--api.insecure=true"
+      - "--providers.file=true"
+      - "--providers.file.watch=true"
+      - "--providers.file.directory=/etc/traefik/configs"
+      - "--entrypoints.web.address=:80"
+    ports:
+      - "80:80"
+      - "8080:8080"
+    volumes:
+      - ./traefik:/etc/traefik/configs
+
+  snapshotter:
+    build:
+      dockerfile: ./snapshotter/Dockerfile
+    command: ["/usr/sbin/crond", "-l", "2", "-f"]
```

Comment:
the current system is intended to be automated, great.

we need to have ways to interact manually on the server without waiting for tomorrow 10am.

could you add a makefile allowing to make changes?
feel free to just make aliases to `docker exec snapshotter -it ./switch.sh`.